tests/unit/modules/test_junos.py

```diff
@@ -3,6 +3,7 @@
     :codeauthor: :email:`Rajvi Dhimar <rajvidhimar95@gmail.com>`
 '''
 # Import python libs
+import os
```

Comment:
Can you move this import down? The first import should be the `__future__` line.